### DISCUSSION TITLE
iOS Code Signing Error with manual signing with .yaml

### DISCUSSION THREAD

**Discussion by FrazeColder** [codemagic-ci-cd/codemagic-docs#2887] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I am facing issues with iOS signing when building my app using Codemagic, but the build works perfectly on my local Mac using automatic code signing.

1. Codemagic Configuration: Codemagic is connected to my developer portal, and I have created an iOS certificate directly through Codemagic. I also selected the correct provisioning profile available.
2. Error Message: When trying to build the app in Codemagic, I encounter the following error:

> Code Signing Error: "error: exportArchive "Runner.app" requires a provisioning profile with the Associated Domains, Push Notifications, and Sign in with Apple features."

3.  I’ve double-checked in the Apple Developer portal (developer.apple.com) under "Identifiers and Capabilities," and I can confirm that all the necessary capabilities — Associated Domains, Push Notifications, and Sign in with Apple — are activated for the app’s bundle identifier.

Steps I’ve Taken:
- Codemagic is linked to my Developer Portal, and the provisioning profile is selected.
- I’ve ensured that the required capabilities (Associated Domains, Push Notifications, and Sign in with Apple) are correctly enabled in the Developer portal.

```
name: Initialize keychain
  script: |
    keychain initialize
name: Add certificates to keychain
  script: |
    keychain add-certificates
name: Set up code signing settings on Xcode project
  script: |
    xcode-project use-profiles
name: Install pods
  script: |
    find . -name "Podfile" -execdir pod install ;
name: Flutter build ipa and automatic versioning
  script: |
    flutter build ipa --flavor production --release \
--build-name=1.0.0 \
--build-number=$(($(app-store-connect get-latest-testflight-build-number "$APP_ID") + 1))
--export-options-plist=/Users/builder/export_options.plist
```

What am I missing or doing wrong here?

### Expected results

Working code signing

### Actual results

Code Signing Error: "error: exportArchive "Runner.app" requires a provisioning profile with the Associated Domains, Push Notifications, and Sign in with Apple features."

**Comment by dam-ease** [comment]
Hello @FrazeColder 
Please add the slash before the `--export-options plist=/Users/builder/export_options.plist` as well.
```
name: Flutter build ipa and automatic versioning
  script: |
    flutter build ipa --flavor production --release \
--build-name=1.0.0 \
--build-number=$(($(app-store-connect get-latest-testflight-build-number "$APP_ID") + 1)) \
--export-options-plist=/Users/builder/export_options.plist
```
Also, kindly share your buildID with me here, You can find it under your build details page's 'ID' section.
